Assets/Scripts/Interactable.cs

```diff
@@ -18,18 +23,66 @@ void Start()
     }
 
     public void Init(){
-        this.myRenderer = GetComponent<SpriteRenderer>();
+        this.myRenderers = new List<SpriteRenderer>();
+        if (GetComponent<SpriteRenderer>() != null){
+            this.myRenderers.Add(GetComponent<SpriteRenderer>());
+        }
         this.selected = false;
+        this.cost = cost;
```

Comment:
This is doing nothing - It's just a shorter way of writing `this.cost = this.cost`. If we're setting this in the inspector there's no need to set it here.